corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -36,22 +36,41 @@
     return template.replace('-locid-', loc_id);
 }
 
+var archive_success_message = _.template(
+    '{% trans "You have successfully archived the location <%=name%>" %}'
+);
```

Comment:
This way translators will see the string `You have successfully archived the location <%=name%>` to translate, and it can be rendered in the javascript as needed.  You could probably argue that just naively concatenating the strings would be sufficient in this case, but whatever.